modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
```

Comment:
This section is a little overwhelming with all of the bolded GUI items. It totally makes sense when you're talking about "... the **Details** pane within the **Virtual Machine Overview** tab ...", but I think that you could list the details that are displayed without actually providing bolded GUI items.

```
When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the host name, operating system, time zone, and logged in users.
```

Reply:
Resolved by removing bolded items and limiting the mention of the GUI items.